src/gui/layout/qgslayoutpicturewidget.cpp

```diff
@@ -549,26 +475,19 @@ void QgsLayoutPictureWidget::svgSourceChanged( const QString &source )
   }
 }
 
-void QgsLayoutPictureWidget::rasterSourceChanged( const QString &source )
+void QgsLayoutPictureWidget::setSvgParameters( const QMap<QString, QgsProperty> &parameters )
```

Comment:
Maybe rename that to set SVG dynamic parameters to be consistent across all classes?